Projects/Features/SignupFeature/Sources/AuthenticationCodeViewModel.swift

```diff
@@ -0,0 +1,32 @@
+import BaseFeature
+import Combine
+import DomainModule
+
+final class AuthenticationCodeViewModel: BaseViewModel {
+
+    @Published var schoolID = ""
+    @Published var schoolCode = ""
+
+    @Published var isOnAutoSignup = true
+    @Published var isNavigateCheckSchool = false
+    @Published var isSuccessAuthenticationCode = false
+    var isSigninButtonEnabled: Bool {
+        !schoolCode.isEmpty
+    }
+
+    private let checkSchoolCodeUseCase: any CheckSchoolCodeUseCase
+
+    public init(checkSchoolCodeUseCase: any CheckSchoolCodeUseCase) {
+        self.checkSchoolCodeUseCase = checkSchoolCodeUseCase
+    }
+
+    func verifyAuthCodeButtonDidTap() {
+        guard isSigninButtonEnabled else { return }
+        addCancellable(
+            checkSchoolCodeUseCase.execute(
+                schoolID: schoolID,
+                code: schoolCode)) { [weak self] _ in
+            self?.isNavigateCheckSchool = true
```

Comment:
```suggestion
                schoolID: schoolID,
                code: schoolCode
            )
        ) { [weak self] _ in
            self?.isNavigateCheckSchool = true
```
으로 하는게
<img width="759" alt="스크린샷 2022-10-22 오전 5 33 33" src="https://user-images.githubusercontent.com/74440939/197283990-015db7fc-cb9c-4323-bad3-0ac8c96537a7.png">
의 StyleGuide에 맞는거같아요!